src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.